.idea/codeStyles/Project.xml

```diff
@@ -37,15 +37,6 @@
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value />
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```

Comment:
this shouldn't be deleted

Reply:
I'll revert all the changes in this file, it inadvertently crept on to the stage